packages/logistration/package.json

```diff
@@ -43,18 +43,18 @@
   "devDependencies": {
     "@edx/browserslist-config": "1.1.0",
     "@edx/frontend-build": "12.7.0",
-    "@edx/frontend-platform": "4.0.1",
+    "@edx/frontend-platform": "4.6.0",
     "@testing-library/jest-dom": "5.11.6",
-    "@testing-library/react": "11.2.6",
-    "react": "16.12.0",
-    "react-dom": "16.12.0",
+    "@testing-library/react": "12.1.4",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
     "react-router-dom": "5.2.0",
-    "react-test-renderer": "16.12.0"
+    "react-test-renderer": "17.0.2"
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^4.0.1",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^16.12.0 || ^17.0.0",
+    "react-dom": "^16.12.0 || ^17.0.0",
```

Comment:
nit: could opt for `^17` if we want to allow all major versions of React v17. Shorthand for `^17.0.0`.